app/controllers/tabs_controller.rb

```diff
@@ -11,13 +11,9 @@ def new
 
 	def create
 		#check parameters of tabs
-		@tab = Tab.new
-		@tab.total = params[:tab][:total]
+		@tab = Tab.new tab_params
 		@tab.date_of_purchase = Time.now
-		@tab.user_id = params[:tab][:user_id]
-		@tab.vendor_id = params[:tab][:vendor_id]
-		@tab.item_list = params[:tab][:item_list]
-		@tab.paid
+		@tab.paid ## Is this supposed to be here like this?
```

Comment:
@jonathanpeterwu I need to update this controller to mirror users/vendors so we aren't hard-coding params[:tab][:blahblah] for every field. I'll default tab.paid = false (we might have put the default in the db already actually)